packages/utils/src/lib/reports/generate-md-report-category-section.unit.test.ts

```diff
@@ -69,9 +62,9 @@ describe('categoryRef', () => {
           score: 1,
         },
         'lighthouse',
-      ),
+      ).toString(),
     ).toBe(
-      '- 🟩 [Score Report Performance](#score-report-performance-lighthouse) (_lighthouse_) - **12245**',
```

Comment:
Why do we remove the "-"?

Reply:
The `categoryRef` function returns the content of a single list item. The `-` gets added when the whole list is rendered, it doesn't happen in this function.